### PR TITLE
Fullstack tests vagrant

### PR DESCRIPTION
**Pull request by rafamanzo**
This is a server that simulates the full mezuro services running under one CentOS 7 server. This is useful for integration tests with external services like Colab.

**DO NOT MERGE BEFORE https://github.com/mezuro/packaging/pull/19**

### CODE REVIEW

**Comment by danielkza** [comment]
Looks good. Rebased and merged.

File: fullstack/Vagrantfile [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+  # end
+
+  # Enable provisioning with a shell script. Additional provisioners such as
+  # Puppet, Chef, Ansible, Salt, and Docker are also available. Please see the
+  # documentation for more information about their specific syntax and use.
+  require_relative '../mezuro_informations'
+  config.vm.provision "shell", inline: <<-SHELL
+    # General system update
+    sed -i 's/keepcache=0/keepcache=1/' /etc/yum.conf
+    yum -y install epel-release
```
Comment (diegoamc) [inline fullstack/Vagrantfile:73]: Why do we need to install the `epel-release` package? Is it a requirement for the system upgrade?

Comment (rafamanzo) [inline fullstack/Vagrantfile:73]: EPEL provides lots of non-standard packages, such as nginx.

Comment (diegoamc) [inline fullstack/Vagrantfile:73]: Ok :+1:

**Pull request merged**